JeMPI_Apps/JeMPI_Linker/src/test/java/org/jembi/jempi/linker/LinkerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.jembi.jempi.linker;
+
+import akka.actor.testkit.typed.javadsl.ActorTestKit;
+import akka.actor.typed.ActorRef;
+import org.jembi.jempi.libmpi.LibMPI;
+import org.jembi.jempi.shared.models.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.util.ArrayList;
+import java.util.List;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class LinkerTest {
+
+    @Mock
+    private LibMPI libMPI;
+    @Test
+    public void testUpdateGoldenRecordField() {
```

Comment:
@mahaomolise92 can you add a test or 2 for errors. This may indicate possible error handling required in code

Reply:
@MatthewErispe I will add more unit tests to address this